src/test/java/com/integrivideo/stories/ManageProjectStoryTest.java

```diff
@@ -63,4 +67,27 @@ public void domainCanBeRemovedAtProjectCreationPage() {
         projectSteps.shouldBeOnProjectsPage();
         projectSteps.numberOfProjectShouldBeEqualTo(projectCount + 1);
     }
+
+    @Test
+    public void userOpensNonExistingProjectWithError() {
+        projectSteps.openProjectUrlByUser("doesNotExist", Data.USER_2_EMAIL, Data.USER_2_PASSWORD);
+        projectSteps.validatePageNotFoundError("doesNotExist");
+    }
+
+    @Test
+    public void userCannotOpenProjectOfAnotherUser() {
+        projectSteps.isOnProjectListPage();
+        long projectCount = projectSteps.getProjectCount();
+        projectSteps.openCreateProjectPage();
+        projectSteps.createProject(Data.PROJECT_NAME, Data.PROJECT_DESCRIPTION, Data.PROJECT_DOMAIN1);
```

Comment:
no need to make assertiones here. Also no need to create project with specific names. Just use createProject() method